ddl/placement_policy_test.go

```diff
@@ -974,6 +990,91 @@ func (s *testDBSuite6) TestDatabasePlacement(c *C) {
 	))
 }
 
+func (s *testDBSuite6) TestDropDatabaseGCPlacement(c *C) {
+	clearAllBundles(c)
```

Comment:
That our tests will call the global `infoscyner` and `failpoints`, is it safe to run in the parallel suite?

Reply:
I think it's safe on condition that all placement sql are executed in serial